### DISCUSSION TITLE
中央期中期間以外沒有24小時的唸書場所

### DISCUSSION THREAD

**Discussion by JaYanlloz** [1101-GS4719-C/Python-Community-GS4719-C#2193] [General]
目前聽到台清交成都有24小時可以唸書的地方，中央只有期中期末週才開放24小時k中，就長期待下來的感覺，其實待到11點被趕走的人不少，至少不可謂"沒有人需要24小時的環境"，這對外宿生或系館資源不夠豐富的科系學生屬實是不太方便，是有什麼法規規定嗎？
而且相對其他幾間，中央的地理位置也是真的很孤島，如果說怕浪費電，至少開個小間k中吧...

**Comment by Jinyuu17** [comment]
幾乎很多學校宿舍都有自修室可以使用，而且不會有時間限制，想讀到幾點就讀到幾點。我之前住男九舍時，都在四樓的自習室讀書到很晚，想準備逃離中央這個鬼地方，但我失敗了。雖然這間自修室在宿舍內，但沒有門禁的人可以跟在宿民後面進去，就有24小時的讀書空間囉！

**Comment by forestdew507** [comment]
目前想起來就只有樓上說的宿舍自修室和原 po 提到的系所空間了

**Comment by poyen0806** [comment]
基本上我都直接住在工程五館，雖然很冷門，但其實裡面的生活品質很高，除了提供24小時的讀書空間，還有衛浴設備，更想不到的是晚上12:00後地下室會開放學生進入，使用特定的學生卡即可入場，裡面是寬廣的地下街，到處都是美食，所以別再抱怨中央大學是美食沙漠了，你只是還沒發現新大陸

**Reply by zoo100130** [reply]
說來弔詭，明明工學院院半在工五，有些工院專門辦的額外課程也在工五，但我是工學院學生證卻在下課後刷不進去QQ

**Comment by yuan3x2** [comment]
系館應該都有讀書室，或是宿舍的讀書室，除了人相對少以外，空間規劃也都挺不錯的，也不用擔心被趕，尤其是在宿舍，累了隨時可以回去睡覺挺方便的。

**Comment by SendaiHatsuki** [comment]
電機系館地下一樓有自己的系K，是24小時開放的，但假日沒有開放，而且進去需要刷學生證，但我覺得環境不太好，因為是地下室的緣故讀書感覺很壓抑，K中的環境我認為比較舒服。
